wordpress-mobile/gutenberg-mobile
# Enable default keyboard as default on link picker text fields

**Fixes** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2733
`Gutenberg PR:` https://github.com/WordPress/gutenberg/pull/29087

## To test:

1. Add `Buttons` block or `Image` Block
1. Open link [settings](https://github.com/wordpress-mobile/test-cases/blob/be95fab2c52f92f369bd7ac9b4a0377304e484ab/test-cases/resources/button-link-settings.png)
1. Edit the URL field and add a link
1. Switch on `Open in new tab`
1. Edit `Link Rel` field
1. Type space to add a new link rel
1. **Expect** To see the default keyboard

## Screenshots

Before | After
--- | ---
<kbd><a href="https://user-images.githubusercontent.com/1845482/96280466-4a5ea000-0fd8-11eb-8e8d-926b334381c8.jpeg"><img width="300" src="https://user-images.githubusercontent.com/1845482/96280466-4a5ea000-0fd8-11eb-8e8d-926b334381c8.jpeg"/></a></kbd> | <kbd><a href="https://user-images.githubusercontent.com/3384451/108255305-17fdb480-712a-11eb-97d2-41b74e012c39.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/108255305-17fdb480-712a-11eb-97d2-41b74e012c39.png"/></a></kbd>

## PR submission checklist:
